components/raftstore-v2/src/fsm/store.rs

```diff
@@ -20,28 +28,54 @@ where
     E: KvEngine,
 {
     pub store_id: Option<u64>,
+    /// region_end_key -> region_id
+    pub region_ranges: BTreeMap<Vec<u8>, u64>,
+    /// region_id -> region
+    pub regions: HashMap<u64, Region>,
     /// region_id -> reader
     pub readers: HashMap<u64, ReadDelegate>,
     /// region_id -> tablet cache
     pub tablet_caches: HashMap<u64, CachedTablet<E>>,
+    /// region_id -> `RegionReadProgress`
+    pub region_read_progress: RegionReadProgressRegistry,
 }
 
 impl<E> StoreMeta<E>
 where
     E: KvEngine,
 {
-    pub fn new() -> StoreMeta<E> {
+    pub fn new(vote_capacity: usize) -> StoreMeta<E> {
```

Comment:
what's vote_capacity for?

Reply:
The field thta needs it has been removed. I will remove this.